modules/ppcp-onboarding/src/Exception/RuntimeException.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * The modules Runtime Exception.
+ *
+ * @package WooCommerce\PayPalCommerce\Onboarding\Exception
+ */
+
+declare(strict_types=1);
+
+namespace WooCommerce\PayPalCommerce\Onboarding\Exception;
+
+/**
+ * Class RuntimeException
+ */
+class RuntimeException extends \RuntimeException {
```

Comment:
Is it needed? Maybe just `\RuntimeException` is enough? These exceptions with the same name in different modules can cause confusion and mistakes.

Reply:
No strong feelings one way or the other, having it will allow for exception distinction on a higher level catch, but not having it will make it easier to generalize if necessary (and OnboardingUrl class it's totally generalizable for other use cases). I will change to \RuntimeException.